lms/resources/_js_config.py

```diff
@@ -159,15 +170,28 @@ def enable_content_item_selection_mode(self, form_action, form_fields):
         :param form_fields: the fields (keys and values) to include in the
             HTML form that we'll use to submit the user's chosen document
         :type form_fields: dict
+
+        :raise ConsumerKeyError: if request.lti_user.consumer_key isn't in the DB
```

Comment:
This could raise `ConsumerKeyError` before and it still can now. Added it to the docstring